### DISCUSSION TITLE
All github.io sites not render properly.

### DISCUSSION THREAD

**Discussion by Fred-Wu** [community/community#40054] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Just noticed that all websites published with `github.io` do not render properly. They lost all formats, and became plain html sites now. 

For examples

https://ramanbala.github.io/dynamic-time-series-models-R-INLA/

https://mfasiolo.github.io/sc2-2019/

**Comment by ossanna16** [comment]
Hi there @Fred-Wu and welcome to our community! Thank you for reporting this!
